src/ess.jl

```diff
@@ -187,10 +178,11 @@ function mean_autocov(k::Int, cache::BDAESSCache)
     # compute mean autocovariance
     n = niter - k
     idxs = 1:n
+    idxs2 = k .+ idxs
```

Comment:
No need for broadcasting here:
```suggestion
    idxs2 = (k + 1):niter
```